DFC.ServiceTaxonomy.ApiFunction/OpenAPIDocs/GetAllOccupationsForSkillOpenApi.txt

```diff
@@ -0,0 +1,107 @@
+﻿openapi: 3.0.1
+info:
+  title: Get Occupations For Given Skill
+  description: 'Sample Description for Query'
+  version: 1.0.0
+servers:
+- url: __FunctionAppUrl__
+tags:
+- name: Get Occupations For Given Skill
+  description: Returns all occupation for a given skill
+paths:
+  /Execute/:
+    post:
+      tags:
+      - Skill
+      summary: Get occupations for a given skill.
+      description: Returns occupation for a given skill
+      operationId: GetOccupationsForSkill
+      requestBody:
+        description: enter esco skill uri to retrieve all Occupations for the Skill
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Skill'
```

Comment:
Is this correct?
'#/components/schemas/Skill' is the schema for the 200 response, don't we need a separate schema for the request, as it only accepts a uri?